src/core/storage/storage-actions.ts

```diff
@@ -12,6 +12,7 @@ import {
 import { StorageWalletStatus } from './storage-reducer'
 
 export const SYNC_INTERVAL = 30 * 1000
+export const EXPEDITED_SYNC_INTERVAL = 5000
```

Comment:
This has nothing to do with storage (it deals with the login server), and should go in the account pixie.